whois-api/src/test/resources/testMail/permanentFailureMessageRfc822MultipleRecipients.mail

```diff
@@ -0,0 +1,77 @@
+Return-path: <>
+Envelope-to: test-dbm@ripe.net
+Delivery-date: Fri, 23 Feb 2024 17:29:29 +0200
+Received: from mahimahi.ripe.net ([193.0.19.1])
+	by allealle.ripe.net with esmtps  (TLS1.2) tls TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+	(Exim 4.94)
+	id xxxxxx-0004d4-OF
+	for test-dbm@ripe.net; Fri, 23 Feb 2024 17:29:29 +0200
+Received: from abc1.host.org ([68.232.147.154]:21686)
+	by mahimahi.ripe.net with esmtps  (TLS1.2) tls TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+	(Exim 4.94.2)
+	id xxxxxx-0009P6-50
+	for test-dbm@ripe.net; Fri, 23 Feb 2024 17:29:29 +0200
+Received: from localhost by abc1.host.org;
+  23 Feb 2024 11:29:19 -0400
+Message-Id: <xxxxxx$353m5e@abc1.host.org>
+Date: 23 Feb 2024 11:29:19 -0400
+To: test-dbm@ripe.net
+From: "Mail Delivery System" <noreply@ces.cisco.com>
+Subject: Delivery Status Notification (Failure)
+MIME-Version: 1.0
+Content-Type: multipart/report; report-type=delivery-status; boundary="6KTYe.5maleLdVv.5/WobWlHy4k.7vnm51B"
+Delivered-To: test-dbm@ripe.net
+
+--6KTYe.5maleLdVv.5/WobWlHy4k.7vnm51B
+content-type: text/plain;
+    charset="utf-8"
+Content-Transfer-Encoding: quoted-printable
+
+The following message to <nonexistant@host.org> was undeliverable.
```

Comment:
Does this mean that *only* `nonexistant@ripe.net` was undeliverable? It's also possible that multiple recipients to the same mail provider are all undeliverable

Reply:
If we have 10 recipients in one email and 5 of them are bounced
We will receive a report for those 5 emails, not for all of 10

In this test both are undeliverable but we just received one just report. This should happen, we should receive a report per recipient...but it is good to have as we discussed